include/tbl/cli_cmds.h

```diff
@@ -48,6 +48,18 @@
 #    include "vsb.h"
 #  endif
 
+static void v_matchproto_(cmd_log_func_t)
+cli_debug_sensitive_log(const struct cli* cli, const char * const *av, struct vsb *vsb)
+{
+
+	(void)cli;
+
+	AN(av);
+	AN(vsb);
+
+	VSB_printf(vsb, "%s %s XXXXX", av[1]!=NULL ? av[1] : "(null)",
+	    av[2] != NULL ? av[2] : "(null)");
+}
```

Comment:
See previous comment on the suspicious `#ifdef`. This function does not belong here.

Reply:
fixed